build.gradle

```diff
@@ -107,6 +107,13 @@ jacocoTestReport {
     html.destination file("${buildDir}/reports/jacoco/html")
     xml.destination file("${project.buildDir}/reports/jacoco/test/jacocoTestReport.xml")
   }
+  afterEvaluate {
+    getClassDirectories().from = getClassDirectories().files.collect {
```

Comment:
why we need this for ?

Reply:
just to be at the same page with sonar config which skips that folder

Reply:
we have it in sonar to exclude code coverage so do we need it in pmd ?

Reply:
not pmd, jacoco ;)

Reply:
we have this `property "sonar.coverage.exclusions", "**/config/**"`  configured when sonar will use jacoco reports it will exclude it anyways so I think we don't need the config added in this PR. What are your thoughts ?

Reply:
hm, true. on the other hand, jacoco report is used by other CIs too: codacy and codecov which both are configured for this project